develop/components/pages/TooltipPage.tsx

```diff
@@ -1,22 +1,81 @@
+    const triggerList: Array<RadioListModel<TooltipTrigger>> = [
+        { label: "Click", value: "click" },
+        { label: "Hover", value: "hover" },
+        { label: "Focus", value: "focus" }
+    ];
+    const themeList: Array<RadioListModel<TooltipTheme>> = [
+        { label: "Primary", value: "primary" },
+        { label: "Danger", value: "danger" },
+        { label: "Default", value: "default" },
+        { label: "Light", value: "light" },
+        { label: "Purple", value: "purple" },
+        { label: "Success", value: "success" },
+        { label: "Warning", value: "warning" },
+    ];
+    const positionList: Array<RadioListModel<TooltipPosition>> = [
+        { label: "Top", value: "top" },
+        { label: "Top Left", value: "top-left" },
+        { label: "Top Right", value: "top-right" },
+        { label: "Right", value: "right" },
+        { label: "Right Top", value: "right-top" },
+        { label: "Right Bottom", value: "right-bottom" },
+        { label: "Bottom", value: "bottom" },
+        { label: "Bottom Left", value: "bottom-left" },
+        { label: "Bottom Right", value: "bottom-right" },
+        { label: "Left", value: "left" },
+        { label: "Left Top", value: "left-top" },
+        { label: "Left Bottom", value: "left-bottom" },
+    ];
+    const defaultMessageGroup: Array<TooltipMessageGroupItem> = [
+        {
+            title: "Tooltip title",
+            message: "tooltip 1"
+        },
+        {
+            message: "tooltip 2"
+        }
+    ];
+    const defaultTooltipContent: string = "Tooltip content could be long, therefore, controlling the position and width is important";
+    const nodeTooltipContent: React.ReactNode = (<div><h1>Tooltip Header</h1><div>tooltip content</div></div>);
+    const [theme, setTheme] = React.useState<TooltipTheme>("primary");
+    const [position, setPosition] = React.useState<TooltipPosition>("top");
+    const [trigger, setTrigger] = React.useState<TooltipTrigger>("click");
+    const [hasCallback, setHasCallback] = React.useState<boolean>(false);
+    const [hasNodeAsContent, setHasNodeAsContent] = React.useState<boolean>(false);
+    const [hasCustomReference, setHasCustomReference] = React.useState<boolean>(false);
+    const [disableAutoPosition, setDisableAutoPosition] = React.useState<boolean>(false);
+    const [customSvg, setCustomSvg] = React.useState<boolean>(false);
+    const [content, setContent] = React.useState<number>(0);
     let myTooltip: Tooltip;
 
     function dismissTooltip(e?: React.MouseEvent<HTMLDivElement>): void {
-        const dismissableTooltip: HTMLElement = document.getElementById("dismissable-tooltip");
-        if (event.target !== dismissableTooltip.firstChild) {
-            myTooltip.forceDismiss(e);
-        }
+        console.log("force dismissed");
+        myTooltip.forceDismiss(e);
+    }
+
+    function forceShowTooltip(e?: React.MouseEvent<HTMLDivElement>): void {
+        console.log("force showed");
```

Comment:
I was thinking to retain the console log as part of the example

Reply:
The documentation should not through console logs that are meant for the user. Since it won't strike the user to check the console for output messages. After we add a toast component, maybe we can consider showing these messages in a toast notification.